src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -34,6 +34,7 @@ menu:
         site_credentials: Accès aux sites
         ignore_origin_instance_rules: "Règles globales d'omission d'origine"
         quickstart: "Pour bien débuter"
+        theme_toggle: "Changer le thème"
```

Comment:
Could you update the `messages.en.yml` too? Because it's the main source language in weblate